File: WebServisi/AutoservisRKS_API/ModelVM/UpitiResultVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace AutoservisRKS_API.ModelVM
{
    public class UpitiResultVM
    {
        public class Row
        {
            public int UpitID { get; set; }
            public string MarkaAuta { get; set; }
            public int KlijentID { get; set; }
            public string DatumSlanja { get; set; }

        }
        public List<Row> rows;
    }
}

File: app/src/main/java/com/example/amarhumaki/autoservis/data/UpitiResultVM.java
package com.example.amarhumaki.autoservis.data;

import java.io.Serializable;
import java.util.List;

public class UpitiResultVM implements Serializable{

    public static class Row implements Serializable {
        public int UpitID;
        public String MarkaAuta; // atribut marka je i za model i marku
        public String KlijentID;
        public String DatumSlanja;
    }
    public List<Row> rows;

}
